app/assets/javascripts/discourse/widgets/post-admin-menu.js.es6

```diff
@@ -64,7 +64,9 @@ export function buildManageButtons(attrs, currentUser) {
       action: 'changePostOwner',
       className: 'change-owner'
     });
+  }
 
+  if (currentUser.staff) {
```

Comment:
Didn't add this to the above `if currentUser.staff` block because I do not want the position of this button to change.